app/assets/stylesheets/application.scss

```diff
@@ -2,3 +2,4 @@
 // = require_self
 // = require select2
 // = require select2-bootstrap
+@import "govuk_publishing_components/all_components";
```

Comment:
To an outsider, it looks like we pull in the CSS from all components in govuk_publishing_components, even if we only use one or two of the components. Is that what happens or is there some post-processing step somewhere?

(I'm sure your code is fine though!)

Reply:
We're going to end up using a lot of the components when we switch to the design system, so we might as well import them all, rather than dragging them in one by one.